src/pixelsquiz/buzz.clj

```diff
@@ -17,18 +17,19 @@
   (let [bits (System/getProperty "sun.arch.data.model")]
     (clojure.lang.RT/loadLibrary (str "hidapi-jni-" bits))))
 
-
 (defn open-buzz []
+  (logger/info "Trying to open Buzz controllers")
```

Comment:
Constantly logging this message (and the other ones below) makes testing more difficult. During testing it's likely that no (physical) controllers are connected.

Reply:
yep. this was added during the last few days before the conf (the stressful times). I was meant to remove it, must have forgotten. fixed now